com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/api/impl/OperationsApiServiceImpl.java

```diff
@@ -84,9 +84,9 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(OperationsApiServiceImpl.class);
 
     @Override
-    public ResponseEntity<List<Operation>> getOperations(String spaceGuid, List<String> stateStrings, Integer last) {
+    public ResponseEntity<List<Operation>> getOperations(String spaceGuid, String mtaId, List<String> stateStrings, Integer last) {
         List<Operation.State> states = getStates(stateStrings);
-        List<Operation> operations = filterByQueryParameters(last, states, spaceGuid);
+        List<Operation> operations = filterByQueryParameters(mtaId, last, states, spaceGuid);
```

Comment:
Could you put it as last param? For me, it is more intuitevly to add new method parameters in the last positions?

Reply:
sure